### ISSUE TITLE
Introduce a new option for workers chunks' filenames

### ISSUE THREAD

**Issue by gretzkiy** [webpack/webpack#17002]
## Feature request

The source of this feature request is this [discussion](https://github.com/webpack/webpack/discussions/16843). The aim is to specify the filename for worker chunks only. Currently, there is no straightforward way to determine whether the chunk being processed in `chunkFilename` is a worker. Therefore, the request is for an option similar to [`workerPublicPath`](https://github.com/webpack/webpack/pull/16671), but only for the worker chunk's filename, such as `workerChunkFilename`.

During the aforementioned discussion, @alexander-akait suggested identifying worker chunks by checking if `chunkLoading === 'import-scripts'` in `pathData?.chunk?._groups` within the `chunkFilename` function. However, this approach is considered hacky and not very robust.

**Comment by TheLarkInn** [comment]
So my understanding is that the `output.workerPublicPath` feature is not enough?

**Comment by alexander-akait** [comment]
@TheLarkInn No, it only set public path for workers, but sometimes you need another location for workers than `dist`, right now you can implement this using `chunkFilename: (pathData) => { /* ... */ }`, but it is really hard to undestand when chunk is worker's chunks when not, so it is more like about user experience
